tests/filecheck/interpreters/pdl_definition.mlir

```diff
@@ -0,0 +1,16 @@
+pdl.pattern : benefit(1) {
```

Comment:
Can you add a comment here to say that this is used in the other test?
Just to not forget that this file is actually used.